src/consensus/steps.ml

```diff
@@ -0,0 +1,185 @@
+open Helpers
+open Crypto
+open Protocol
+open State
+open Is_signable
+
+type signed =
+  | Signed
+  | Not_signed
+
+type step =
```

Comment:
why not type t?

Reply:
https://github.com/marigold-dev/deku/pull/674#discussion_r893895978